be/src/common/version.h

```diff
@@ -21,4 +21,5 @@ extern const char* STARROCKS_BUILD_TYPE;
 extern const char* STARROCKS_BUILD_TIME;
 extern const char* STARROCKS_BUILD_USER;
 extern const char* STARROCKS_BUILD_HOST;
+extern const char* STARROCKS_BUILD_DISTRO_ID;
```

Comment:
`STARROCKS_BUILD_DISTRO` is a better name?

Reply:
refer to lsb_release output

```
$ lsb_release -a
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	CentOS
Description:	CentOS Linux release 7.9.2009 (Core)
Release:	7.9.2009
Codename:	Core
```

Reply:
Is the `CentOS-7.9.2009` better than `CentOS`?

Reply:
yes it is absolutely better for human readable. However, it might not be friend to script. `id` field is a much stable and fixed field for scripting.

I still opt to use the `id` for our case. @chaoyli

Reply:
How about output `Description` like `CentOS Linux release 7.9.2009 (Core)`?  @kevincai

Reply:
we can add additional `Description` line in the output. but the id should be as simple as possible.